types/url-parse/url-parse-tests.ts

```diff
@@ -0,0 +1,10 @@
+import UrlParse = require("url-parse");
+
+const parse = UrlParse.default;
```

Comment:
This is not the usage shown in the docs. You should be using `export = ` rather than `export default` in index.d.ts

Reply:
Thanks for the hint! 
Will change it